tests/saitests/sai_qos_tests.py

```diff
@@ -2198,12 +2215,24 @@ def runTest(self):
         buf_pool_roid=int(self.test_params['buf_pool_roid'], 0)
         print >> sys.stderr, "buf_pool_roid: 0x%lx" % (buf_pool_roid)
 
+        buffer_pool_wm_base = 0
+        if 'cisco-8000' in asic_type:
+            # Some small amount of memory is always occupied
+            buffer_pool_wm_base = sai_thrift_read_buffer_pool_watermark(self.client, buf_pool_roid)
```

Comment:
So looks like some buffers are researved? What's the expected value, should add an assert to check here?